machine/minit.c

```diff
@@ -169,7 +169,7 @@ static void hart_plic_init()
   for (size_t i = 0; i < ie_words; i++) {
```

Comment:
The `ie_words` calculation is now wrong. Let's change it to use `sizeof(*HLS()->plic_s_ie)`?